generator/osm2meta.hpp

```diff
@@ -96,6 +97,7 @@ class MetadataTagProcessor : private MetadataTagProcessorImpl
     case Metadata::FMD_BANNER_URL: valid = ValidateAndFormat_url(v); break;
     case Metadata::FMD_LEVEL: valid = ValidateAndFormat_level(v); break;
     case Metadata::FMD_AIRPORT_IATA: valid = ValidateAndFormat_airport_iata(v); break;
+    case Metadata::FMD_DURATION: valid = ValidateAndFormat_duration(v); break;
```

Comment:
может указать к чему относится этот duration в названии?

Reply:
```cpp
FMD_LEVEL = 28,
FMD_AIRPORT_IATA = 29,
FMD_BRAND = 30,
// Because there are small amount of ferries with duration,
// we use Metadata to store this tag.
FMD_DURATION = 31,
FMD_COUNT
```

Reply:
Это могут быть не только паромы.
```
// Duration of routes by ferries and other rare means of transportation.
// The number of ferries having the duration key in OSM is low so we
// store the parsed tag value in Metadata instead of building a separate section for it.
// See https://wiki.openstreetmap.org/wiki/Key:duration
```

Reply:
done